frontend/app/(footer)/error/page.tsx

```
@@ -1,13 +1,56 @@
 "use client"; // Error components must be Client Components
 
+import { HighlightButton } from "@/components/elements/HighlightButton";
+import { ArrowLeftIcon } from "@heroicons/react/24/outline";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter, useSearchParams } from "next/navigation";
+
 export default function Error() {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)